ENGAGEHF/Home.swift

```diff
@@ -78,7 +78,7 @@ struct HomeView: View {
                 AccountSheet()
             }
             .accountRequired(!FeatureFlags.disableFirebase && !FeatureFlags.skipOnboarding) {
-                AccountSetupSheet()
+                AccountSetup()
```

Comment:
I think this should move back to the AccountSetupSheet to show the invitation code flow + login.